test/README.md

```diff
@@ -0,0 +1,54 @@
+# Detailed test-fixture build steps
+
+The following steps will attempt to outline the current build process for the Expo test fixture contained within this repo for future reference.
+
+## Requirements
+
+The requirements for a machine building the test fixture on CLI can be found within the `platforms-ansible` repository.
```

Comment:
Worth mentioning it's private so that external viewers don't waste time looking for it?

```suggestion
The requirements for a machine building the test fixture on CLI can be found within our private `platforms-ansible` repository.
```